taxcalc/tests/test_calculate.py

```diff
@@ -117,14 +118,21 @@ def test_make_Calculator_deepcopy():
 
 
 def test_make_Calculator_from_files(paramsfile):
```

Comment:
I still advocate for a way to make a Parameters object from a file name, but I would actually kill this test since there is no longer a `from_files` class method on `Calculator`